pkg/server/registry/service.go

```diff
@@ -15,20 +15,27 @@ import (
 )
 
 type LocalContainerRegistryConfig struct {
-	DataPath string
-	Port     uint32
+	DataPath      string
+	Port          uint32
+	RegistryImage string
```

Comment:
I suggest renaming this to `Image` since it's clear it refers to a registry image because it's a config prop of the ContainerRegistry.